packages/design-preview/src/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const COLOR_VARIATIONS_BLACK_LIST = [
```

Comment:
[nit] Shall we use another name for this? My suggestions:
- COLOR_VARIATIONS_DENY_LIST
- COLOR_VARIATIONS_DISALLOW_LIST
- COLOR_VARIATIONS_BLOCK_LIST